### PR TITLE
Update mono workload id and version to match the workload spec

### PR DESCRIPTION
**Pull request by steveisok**
The id of the package we produce will be `Microsoft.NET.Workload.Mono.ToolChain.Manifest-<sdk-band>`

### CODE REVIEW

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj [hunk 1]
```diff
@@ -6,12 +6,30 @@
   </PropertyGroup>
 
   <Target Name="_PrepareForPack" BeforeTargets="GetPackageFiles" Returns="@(PackageFile)">
+    <!-- Override the id to include the sdk band as per the workload installer spec -->
+    <PropertyGroup>
+      <SdkBandVersion>6.0.100</SdkBandVersion>
```
Comment (marek-safar) [inline src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj:11]: Should the version be hardcoded here?

Comment (steveisok) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj:11]: I think so, temporarily.

It's my understanding that when the sdk band bumps (say to 6.0.200), we will need to publish a new baseline package.  I'm wondering if we need to place this in a new repo for that reason alone?  That is, unless we release a new runtime with each sdk bump.

Id like to get the right package out to the sdk now and shift to a new repo, if necessary, shortly after.

Comment (marek-safar) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj:11]: If the file stays here I think the version should go to runtime/eng/Versions.props

**Comment by steveisok** [comment]
Official build run https://dev.azure.com/dnceng/internal/_build/results?buildId=1129164&view=results

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj (hunk 1 above)
Comment (steveisok) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/Microsoft.NET.Workload.Mono.Toolchain.Manifest.pkgproj]: Sounds good to me.

**Review by akoeplinger — APPROVED** [review]

**Review by lewing — APPROVED** [review]

**Pull request merged**